source/rst/continuous_time/seir_model_sde.rst

```diff
@@ -0,0 +1,914 @@
+
+The notation is standard in the epidemiology literature - though slightly confusing, since :math:`R(t)` is different to
+:math:`R`, the symbol that represents the removed state. Throughout the rest of the lecture, we will always use :math:`R` to represent the effective reproduction number, unless stated otherwise.
+
+Assume that there is a constant population of size :math:`N` throughout, then define the proportion of people in each state as :math:`s := S/N` etc.  With this, and assuming a continuum approximation, the SEIR model can be written as
+
+.. math::
+   \begin{aligned}
+        \frac{d s}{d t}  & = - \gamma \, R \, s \,  i
+        \\
+        \frac{d e}{d t}   & = \gamma \, R \, s \,  i  - \sigma e
+        \\
+         \frac{d i}{d t}  & = \sigma  e  - \gamma i
+        \\
+         \frac{d r}{d t}  & = \gamma  i
+   \end{aligned}
+   :label: seir_system
+
+Here, :math:`dy/dt` represents the time derivative for the particular variable.
+
+Since the states form a partition, we could reconstruct the "removed" fraction of the population as :math:`r = 1 - s - e - i`.  However, for further experiments and plotting it is harmless to keep it in the system.
+
+
+Since we are using a continuum approximation, all individuals in the population are eventually infected when
+the transmission rate is positive and :math:`i(0) > 0`.
+
+We can begin writing the minimal code to solve the dynamics from a particular ``x_0 = [s_0, e_0, i_0, r_0]`` initial condition and parameter values.
+
+First, define the system of equations
+
+.. code-block:: julia
+
+    function F_simple(x, p, t; γ = 1/18, R = 3.0, σ = 1/5.2)
+        s, e, i, r = x
+
+        return [-γ*R*s*i;       # ds/dt = -γRsi
+                 γ*R*s*i -  σ*e;# de/dt =  γRsi -σe
+                 σ*e - γ*i;     # di/dt =        σe -γi
+                       γ*i;     # dr/dt =            γi
+                ]
+    end
+
+Written this way, we see that the four equations represent the one-directional transition from the susceptible to removed state, where the negative terms are outflows, and the positive ones inflows.  The equation associated with the state :math:`R` has inflows and no outflows, so any agents which end up in that state are "trapped".
+
+Consequently, with no flow leaving the :math:`dr/dt` and strictly positive parameter values the ``R`` state is absorbing, and :math:`\lim_{t\to \infty} r(t) = 1`.  Crucial to this result is that individuals are perfectly divisible, and any arbitrarily small :math:`i > 0` leads to a strictly positive flow into the exposed state.
+
+We will discuss this topic (and related topics of irreducibility) in the lecture on continuous-time Markov chains, as well as the limitations of these approximations when the discreteness becomes essential (e.g. continuum approximations are incapable of modeling extinguishing of an outbreak).
```

Comment:
Perhaps this is not true once aggregate shocks are added, if, say, the paths admit jumps.  Might be worth qualifying?

Reply:
I spent a bunch of time looking into this and clearing this up.  This turns out to be a very subtle point, but what I had written before was correct.  It would only be true if R_0 > 0 for this sort of model, I think.

The latest updates though have changed a lot so this won't be an issue.